managed-ledger/src/main/java/org/apache/bookkeeper/mledger/util/StatsBuckets.java

```diff
@@ -87,6 +98,16 @@ public double getAvg() {
         return sum / (double) count;
     }
 
+    public void addAll(StatsBuckets other) {
+        checkArgument(boundaries.length == other.boundaries.length);
```

Comment:
since the `other` is also a `StatsBuckets` the sorted check would have been already performed in its own constructor